ts/features/pushNotifications/store/reducers/__tests__/userBehaviour.test.ts

```diff
@@ -20,4 +25,31 @@ describe("userBehaviourReducer", () => {
     );
     expect(userBehaviourState.engagementScreenShown).toBe(true);
   });
+  it(' "pushNotificationsBanner.timesDismissed" should be "1" after receiving "setUserDismissedNotificationsBanner"', () => {
+    const userBehaviourState = userBehaviourReducer(
+      undefined,
+      setUserDismissedNotificationsBanner()
+    );
+    expect(userBehaviourState.pushNotificationsBanner.timesDismissed).toBe(1);
+  });
+  it(' "pushNotificationsBanner.forceDismissionDate" should be "Date" after receiving "setPushNotificationBannerForceDismissed"', () => {
+    const userBehaviourState = userBehaviourReducer(
+      undefined,
+      setPushNotificationBannerForceDismissed()
+    );
+
+    expect(
```

Comment:
If we convert it to a number:
expect(typeof userBehaviourState.pushNotificationsBanner.forceDismissionDate).toBe("number")